src/emc/plots/spatial.py

```diff
@@ -0,0 +1,136 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import Normalize
+from matplotlib import rcParams, ticker, cm
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.colors as mcolors
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+
+
+def _global(latitude, longitude, data, **kwargs):
```

Comment:
if you are going to use `**kwargs`, you should check if that arg is present when you use it.
If not present, it should provide a default or error out.
Also, instead of `**kwargs`, I suggest with pass a dictionary.  The dictionary is empty by default.

Reply:
```suggestion
def _global(latitude, longitude, data, plotopts=dict()):
```

Reply:
@kevindougherty-noaa and I talked in length about the dictionary vs args issue in these functions.

I agree it probably needs to check if it exists. However, he convinced me that it is more work/less clean to have a dictionary of options passed into these functions than just making them optional arguments.

Reply:
you could have both, `plotopts=dict()` and `**kwargs`

Reply:
The rationale for a dictionary comes from this:
https://github.com/aerorahul/pyarsenal/blob/develop/src/pyarsenal/specs.py

We can store a database of `field_specs`, that populates and gives a default dictionary of values.